src/Smp.Web/Models/Requests/RequestRequest.cs

```diff
@@ -5,11 +5,8 @@ namespace Smp.Web.Models.Requests
 {
     public class RequestRequest
     {
-        [Required]
-        public Guid SenderId { get; set; }
-        [Required]
-        public Guid ReceiverId { get; set; }
-        [Required]
-        public byte RequestTypeId { get; set; }
+        [Required] public Guid SenderId { get; set; }
```

Comment:
Please pick one style. Having attributes on a separate line above the property seems to be the most used way of doing this.

Reply:
Notice how those on a separate lines are way longer, therefore the lines would exceed the 120 character limit enforced by the style checker. I'll revert to having the attributes on a separate line and amend the style checker to enforce this style in the future.

Reply:
Current values:
<img width="1094" alt="image" src="https://user-images.githubusercontent.com/42574496/74871236-0afd8800-5353-11ea-85ad-3a66ae8c5adb.png">

Reply:
Addressed in 9902032.